src/v/pandaproxy/rest/handlers.cc

```diff
@@ -35,7 +36,10 @@
 #include "pandaproxy/parsing/httpd.h"
 #include "pandaproxy/reply.h"
 #include "pandaproxy/rest/configuration.h"
+#include "pandaproxy/sharded_client_cache.h"
 #include "raft/types.h"
+#include "security/credential_store.h"
+#include "security/scram_algorithm.h"
```

Comment:
Are these includes needed?

Reply:
Ah not anymore now that authentication is only in the request_auth.h 
I'll remove.